### PR TITLE
Put images and colors in own module

### LINKED ISSUE #1719: Move all images and colors to own module
To make it easier to use specialized images and colors, we should move all of these into its own module, which can then be replaced or extended.

### PR DESCRIPTION
**Pull request by LosD**
This will put images and colors in a new module.

Images are made available as paths from properties files and through `ResourceBundle`s. Colors are only available through `ResourceBundle`s.

Please note: Original images colors has _not_ been removed yet, and the images and colors aren't actually used yet. This will come later

Fixes #1719

### CODE REVIEW

File: oss-branding/src/main/java/ImageLoadingPropertyResourceBundle.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This loads images from a property file containing resource paths.
+ */
+public class ImageLoadingPropertyResourceBundle extends ResourceBundle {
+    private static final Logger logger = LoggerFactory.getLogger(ImageLoadingPropertyResourceBundle.class);
+
+    private final Map<String, String> paths;
```
Comment (jhorcicka) [inline oss-branding/src/main/java/ImageLoadingPropertyResourceBundle.java]: Missing underscore?

**Comment by jhorcicka** [comment]
Build fails because of checkstyle, mostly missing EOLs.

**Comment by jhorcicka** [comment]
When I try to run it (after building without tests), I get: 
```
ERROR 13:16:12 DCUncaughtExceptionHandler - Thread main threw uncaught exception
java.lang.ExceptionInInitializerError
	at org.datacleaner.util.LookAndFeelManager.init(LookAndFeelManager.java:102)
	at org.datacleaner.bootstrap.Bootstrap.runInternal(Bootstrap.java:166)
	at org.datacleaner.bootstrap.Bootstrap.run(Bootstrap.java:102)
	at org.datacleaner.Main.main(Main.java:165)
	at org.datacleaner.Main.main(Main.java:150)
Caused by: java.lang.IllegalStateException: java.awt.FontFormatException: java.nio.BufferUnderflowException
	at org.datacleaner.util.WidgetUtils.createFont(WidgetUtils.java:332)
	at org.datacleaner.util.WidgetUtils.<clinit>(WidgetUtils.java:119)
	... 5 more
Caused by: java.awt.FontFormatException: java.nio.BufferUnderflowException
	at sun.font.TrueTypeFont.init(TrueTypeFont.java:586)
	at sun.font.TrueTypeFont.<init>(TrueTypeFont.java:198)
	at sun.font.TrueTypeFont.<init>(TrueTypeFont.java:178)
	at sun.font.SunFontManager.createFont2D(SunFontManager.java:2461)
	at java.awt.Font.<init>(Font.java:615)
	at java.awt.Font.createFont0(Font.java:969)
	at java.awt.Font.createFont(Font.java:877)
	at org.datacleaner.util.WidgetUtils.createFont(WidgetUtils.java:330)
	... 6 more
```

**Comment by LosD** [comment]
Weird with the EOL and EOF. Thought I set it up to do that automatically... Oh well

**Comment by LosD** [comment]
I think that should do it

**Comment by jhorcicka** [comment]
It is buildable and runnable now. ;-)

**Comment by jhorcicka** [comment]
LGTM.

**Pull request merged**